week2/2-website/css/style.css

```diff
@@ -1,19 +1,261 @@
+    display: flex;
+    text-decoration: none;
+    border: 1px solid red;
+    justify-content: space-around;
+    }
+    nav-content {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    align-items: centerjustify;
+    padding: 0.5rem;
+    height: 5rem;
+    margin: 0.5rem;
+    
+    }
+    nav-list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style-type: none;
+    font-size: 1.1rem;
+    padding: 0.5rem;
+    margin:0.3rem;
+    list-style : none;
+}
+    nav a {
+        text-decoration: none;
+        color: rgb(226, 213, 213);
+    
+    }
+
+    .links {
+        list-style:none;
+        display: flex;
+        flex-direction: row;
+        align-items: flex-end;
+        
+    }
+
+.nav-link {
+
+    padding: 3rem;
+}
+    #logo {
+    height: 2.75rem;
+    width: 2.75rem;
+    margin-right: 9rem;
+    }
+    .introduction {
+        background: url(../img/first-background.jpg)no-repeat center;
+        padding: 25%;
+        color: white;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
+        background-size: cover;
+    }
+    .intro-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    }
+    h1
+    {
+    color: black;
+    font-size: 3rem;
+    font-weight: lighter;
+    margin-top: 15rem;
+    }
+    p {
+    color: black;
+    font-size: 1.5rem;
+    font-weight: lighter;
+    }
+    button {
+    
+    text-align: center;
+    font-size: 1.8rem;
+    border-radius: 0.2rem;
+    border: none;
+    width:12rem;
+    padding:1rem;
+    margin: 2rem;
+    }
+    button:hover {
+    background-color: rgb(247, 68, 3);
+    }
+    .icons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    }
+    .icons-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    }
+    h2 {
+    text-align: center;
+    font-size: 3rem;
+    font-weight: lighter;
+    margin-top: 8rem;
+    }
+    .icon-list{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    list-style-type: none;
+    }
+    .icon-list li {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+    margin: 2rem;
+    box-sizing: border-box;
+    text-align: center;
+    }
+    .img-icon {
+    width: 75%;
+    height: 75%
+    }
+    .icon-h4 {
+    font-size: 1.5rem;
+    font-weight: lighter;
+    margin: 0rem;
+    padding: 0rem;
+    }
+    footer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    }
+    hr {
+    width: 70%;
+    texy-align: center;
+    margin-top: 5rem;
+    }
+    h5 {
+    font-size: 1.2rem;
+    font-weight: normal;
+    text-align: center;
+    box-sizing: border-box;
+    }
+    .sm-icons {
```

Comment:
Are there any items with .sm-icons in your html? Otherwise this isn't being used.